.evergreen/csfle/activate_venv.sh

```diff
@@ -21,5 +25,5 @@ fi
 
 # install dependencies on first run
 if [ ! -z $FIRST_RUN ]; then
-  pip install --upgrade boto3~=1.19 pykmip~=0.10.0
-fi
\ No newline at end of file
+  CRYPTOGRAPHY_DONT_BUILD_RUST=1 pip install --upgrade boto3~=1.19 cryptography~=3.4.8 pykmip~=0.10.0
```

Comment:
Out of curiosity, where is the new `CRYPTOGRAPHY_DONT_BUILD_RUST` env variable referenced?

Reply:
It is used during build of the `cryptography` package; see [cryptography 3.4 documentation](https://cryptography.io/en/3.4/faq.html#installing-cryptography-fails-with-error-can-not-find-rust-compiler) and the [latest docs](https://github.com/pyca/cryptography/blob/75be92de8e3bce9adcec42ef3967bed0d4500902/docs/installation.rst#rust) regarding Rust requirement. I don't expect this workaround (version pin + `CRYPTOGRAPHY_DONT_BUILD_RUST`) to be a long-term solution; I think the proper long-term solution would be to install the Rust toolchain on all necessary Evergreen variants, but that is beyond the scope of changes I require to unblock related work.

Reply:
SGTM; thanks for the context.